### PR TITLE
Bug fixes

### LINKED ISSUE #359: parse adds Mapping fields to MappingSet header
In my mapping file (`obo_omop.tsv`) I am using the "comment" field to add notes for some of the mappings.  I have the mapping set level metadata in a separate `metadata.yml` file and it does *not* include the "comment" field.  When I run:
```
sssom parse obo_omop.tsv --metadata metadata.yml -o obo_omop.sssom.tsv
```
the resulting `obo_omop.sssom.tsv` file contains the "comment" field as specified in `obo_omop.tsv`, but it also adds "comment" to the MappingSet header along with the last value from the comment field in `obo_omop.tsv`.

I did some experimenting and found that this issue occurs for any of the fields from the MappingSet class that have the same name as a field in the Mapping class (e.g., comment, mapping_date, object_source, subject_source, other, etc.).  So, if you use one of these common name fields in your mapping file and don't use it in the metadata file, then `parse` will add the field to the mapping set header.

### PR DESCRIPTION
**Pull request by hrshdhgd**
- [x] If `predicate_modifier` absent in a `sssom` file, `invert_mappings` errors
- [x] Fixed #359

### CODE REVIEW

**Review by matentzn — DISMISSED** [review]

File: sssom/util.py [hunk 1]
```diff
@@ -1542,6 +1542,8 @@ def invert_mappings(
     :param predicate_invert_dictionary: YAML file providing the inverse mapping for predicates.
     :return: Pandas dataframe with all subject IDs having the same prefix.
     """
+    if PREDICATE_MODIFIER not in df.columns:
+        df["predicate_modifier"] = ""
```
Comment (matentzn) [inline sssom/util.py:1546]: Remind me why this has to be here? Will the column removed if this turns out empty?

**Review by matentzn — DISMISSED** [review]
One small question but otherwise looks great.

File: sssom/util.py (hunk 1 above)
Comment (matentzn) [reply sssom/util.py:1546]: Will the column appear in the inverted data frame even if its empty?

Comment (hrshdhgd) [reply sssom/util.py:1546]: for now , yes. But sometimes even if the column exists aren't the chances of it being completely empty high?

Comment (matentzn) [reply sssom/util.py:1546]: I would prefer if the column is not added if it didn't exist earlier. Is this difficult to achieve?

Comment (hrshdhgd) [reply sssom/util.py]: Fixed.

**Review by matentzn — APPROVED** [review]
Excellent looks good to me!

**Pull request merged**